### DISCUSSION TITLE
compile_container

### DISCUSSION THREAD

**Discussion by blackstormlab** [DirectoryLister/DirectoryLister#1242] [Help & Support]
what is compile_container and why do i need to disable it when running nginx with php in docker ?
am i missing any functionalty by disabling it ?

**Comment by PHLAK** [comment]
> Control whether or not the container is compiled.

https://docs.directorylister.com/configuration/app-config-reference#compile_container

You should _not_ need to modify this setting for Directory Lister to work under normal circumstances. Disabling it is only necessary when running Directory Lister on a read-only file system (see [here](https://github.com/DirectoryLister/DirectoryLister/issues/613) for context).

What errors are you seeing when it's enabled?

**Reply by blackstormlab** [reply]
I ended u figuring out it was a permission issue even though the container is supposed to set them up correctly on start.

**Reply by blackstormlab** [reply]
Aka it was basically acting as a read only file system.
